andriy.safronov/01.rb

```diff
@@ -0,0 +1,3 @@
+r=Array.new(3){ Array.new(3) {rand (-10..10)}}.flatten.select { |r| r>0 }
```

Comment:
```suggestion
r = Array.new(3){ Array.new(3) { rand (-10..10) } }.flatten.select { |r| r>0 }
```

https://github.com/rubocop-hq/ruby-style-guide#spaces-operators
It's better to add spaces so every part of your code is stand out visually

Reply:
Thank you.